pkg/lib/machines.js

```diff
@@ -570,6 +573,30 @@
                     });
             }
 
+            /* Try to get change notifications via the internal
+               /packages D-Bus interface of the bridge.  Not all
+               bridges support this API, so we still get the first
+               version of the manifests via HTTP in request_manifest.
+            */
+
+            function watch_manifests() {
+                var dbus = cockpit.dbus(null, { bus: "internal",
```

Comment:
Hmm, I am no longer certain combining the echo channel and the d-bus watching is worth it.  Right now, the roles are nicely separated.